docs/design/design-reviews/recipes/engagement-process.md

```diff
@@ -12,7 +12,7 @@ The following guidelines outline how Microsoft and the customer together can inc
 
 ## Envisioning / Architecture Design Session (ADS)
 
-Early in an engagement Microsoft works with customers to understand their unique goals and objectives and establish a definition of done. Microsoft dives deep into existing customer infrastructure and architecture to understand potential constraints.
+Early in an engagement Microsoft works with customers to understand their unique goals and objectives and establish a definition of done. Microsoft dives deep into existing customer infrastructure and architecture to understand potential constraints. Recommend leveraging [Non-Functional Requirements Guidance](../../design-patterns/non-functional-requirements-capture-guide.md) to assist.
```

Comment:
Nit: 
```suggestion
Early in an engagement Microsoft works with customers to understand their unique goals and objectives and establish a definition of done. Microsoft dives deep into existing customer infrastructure and architecture to understand potential constraints. Additionally, we seek to understand and uncover specific [non-functional requirements](../../design-patterns/non-functional-requirements-capture-guide.md) that the customer might have.
```

Reply:
Re-worked; a mix of your approach and further editorial license from my end :)